docs/UserGuide.md

```diff
@@ -148,10 +148,20 @@ You may check out the summary table of commands for the overview.
 
 Shows a message explaining how to access the help page. You may use this frequently when you first begin using d'Interiéur.
 
+When you're feeling overwhelmed by the number of commands available, refer to the [command summary](#command-summary) and the examples!
```

Comment:
I thought this was really great considering how as a reader, I might feel lost when there are so many features. A little check in like that would be nice 😃